src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -235,6 +235,15 @@ export default class rd2EntryForm extends LightningElement {
      * @param event
      */
     handleElevateWidgetDisplayState(event) {
+
+        const shouldResetPaymentMethodInEditMode = this.isEdit
```

Comment:
could we extract a function from this constant instead of having the block inside the `handle` function?